src/Compilers/CSharp/Portable/Binder/Binder_Symbols.cs

```diff
@@ -924,7 +925,11 @@ private NamedTypeSymbol BindPredefinedTypeSymbol(PredefinedTypeSyntax node, Bind
             }
 
             result.Free();
-            return NamespaceOrTypeOrAliasSymbolWithAnnotations.CreateUnannotated(AreNullableAnnotationsEnabled(node.Identifier), bindingResult);
+
+            var namespaceOrTypeOrAlias = NamespaceOrTypeOrAliasSymbolWithAnnotations.CreateUnannotated(AreNullableAnnotationsEnabled(node.Identifier), bindingResult);
+            ReportFieldOrValueContextualKeywordConflicts(node, node.Identifier.Text, namespaceOrTypeOrAlias.Symbol, diagnostics);
```

Comment:
Is it possible for the symbol to be the accessor `value` parameter symbol here, or would passing `symbol: null` here work?

Reply:
No, I don't think the symbol can be the `value` parameter here.